### PR TITLE
[Refactor] Removed Colors.navigationTitleColor

### PR DESCRIPTION
**Pull request by eviltofu**
# TextField
Light mode | Dark mode
-|-
TextField-light|TextField-dark

# TokenCardRowViewModelProtocol
Light mode | Dark mode
-|-
icon-light|icon-dark

# ImportMagicTokenViewControllerViewModel
Light mode | Dark mode
-|-
import-light|import-dark

### CODE REVIEW

File: AlphaWallet/Common/Views/TextField.swift
Comment (hboon) [inline AlphaWallet/Common/Views/TextField.swift:344]: Capital B in `navigationbarPrimaryFont`? Also seems like the semantic name is wrong. It's a UIButton right?

File: AlphaWallet/Common/Views/ViewModels/TokenCardRowViewModelProtocol.swift
Comment (hboon) [inline AlphaWallet/Common/Views/ViewModels/TokenCardRowViewModelProtocol.swift:49]: Is this symbol as in a token (eg. ERC20's) symbol? Otherwise maybe `.icon` would be better?

Comment (eviltofu) [reply AlphaWallet/Common/Views/ViewModels/TokenCardRowViewModelProtocol.swift:49]: There is already a different color named icon.

Comment (hboon) [reply AlphaWallet/Common/Views/ViewModels/TokenCardRowViewModelProtocol.swift:49]: Sure

File: AlphaWallet/Common/Views/TextField.swift
Comment (eviltofu) [reply AlphaWallet/Common/Views/TextField.swift:344]: Change this to `.secureIcon`?

**Comment by eviltofu** [comment]
Will squish once approved.

**Review by hboon — APPROVED** [review]

**Comment by hboon** [comment]
IMO, better to squash it in the first place before pushing unless it's really needed. We have had such commits accidentally merged into master without squashing.

**Comment by hboon** [comment]
Added `WIP. Don't Merge` as a reminder. Feel free to remove before merge.

**Pull request merged**
